addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java

```diff
@@ -30,71 +30,62 @@
  *
  * Current state of implementation:
  * <ul>
- * <li>Only EN and DE languages supported</li>
+ * <li>All API languages supported</li>
  * <li>Only default voice supported with good audio quality</li>
- * <li>Only MP3 audio format supported</li>
+ * <li>Only MP3, OGG and AAC audio formats supported</li>
  * <li>It uses HTTP and not HTTPS (for performance reasons)</li>
  * </ul>
  *
  * @author Jochen Hiller - Initial contribution
+ * @author Laurent Garnier - add support for all API languages
+ * @author Laurent Garnier - add support for OGG and AAC audio formats
  */
 public class VoiceRSSCloudImplementation implements VoiceRSSCloudAPI {
 
     private final Logger logger = LoggerFactory.getLogger(VoiceRSSCloudImplementation.class);
 
-    private static Set<String> supportedAudioFormats = getSupportedAudioFormats();
-    private static Set<Locale> supportedLocales = getSupportedLocales();
-    private static Set<String> supportedVoices = getSupportedVoices();
-
-    /**
-     * Will support only "MP3" and "OGG" for the moment.
-     */
-    private static Set<String> getSupportedAudioFormats() {
-        Set<String> formats = new HashSet<String>();
-        formats.add("MP3");
-        formats.add("OGG");
-        return formats;
+    private static Set<String> supportedAudioFormats = new HashSet<String>();
+    static {
```

Comment:
Static initializers looks more readable and simpler to me and are often used in OH / smarthome code.